src/main/java/racingcar/MovingStrategy.java

```diff
@@ -0,0 +1,5 @@
+package racingcar;
+@FunctionalInterface
```

Comment:
import 선언 영역과 실제 코드 영역은 줄바꿈하여 분리해주시면 좋을 것 같습니다. 😃

```suggestion
package racingcar;

@FunctionalInterface
```

Reply:
감사합니다 수정완료했습니다!